Homework6.py

```diff
@@ -0,0 +1,157 @@
+import matplotlib.pyplot as plt
+import random
+plt.style.use('ggplot')
+
+''' Our aim was to create an improved model
+    of bacterial cells growth and death in vitro,
+    taking into consideration inappropriate environment
+    The original model may be checked at the 14 page of
+    "BRS Microbiology and Immunology, 6th edition"
+    This script contains:
+    class Bacteria - to describe bacterial cells
+    class Medium - to describe environment and the experiment
+    class Scientist - to describe and visualize the results
+    and to describe the scientist
+    function go_on - to start and run the experiment properly'''
+
+
+class Bacteria(object):
+
+    exp_data = list()
+    bacteria_count = list()
+
+    def __init__(self, species, risk, bacteria_count):
+        """species - variable to describe the name of bacteria
+
+        risk - variable to describe risk of bacterial
+        death out of inappropriate media
+
+        bacteria_count - contains a list, where each bacteria
+        is an integer [1], so we'll be able to manipulate
+        with each bacteria in future
+
+        """
+
+        self.species = species
+        self.risk = risk
+        self.bacteria_count = list(1 for x in range(bacteria_count))
```

Comment:
overall too many range len etc. more of that do we really need list [1,1,1...] for bacterias? could we just use int?